velox/common/file/benchmark/ReadBenchmark.cpp

```diff
@@ -43,6 +43,12 @@ DEFINE_int32(
     "Total reads per thread when throughput for a --bytes/--gap/--/gap/"
     "--num_in_run combination");
 
+static bool IsNonEmpty(const char* flagname, const std::string& value) {
```

Comment:
Can this function go into an anonymous namespace?

naming: isNotEmpty or notEmpty
naming: flagname -> flagName, but since it is not used it needs to be commented out